WordPress/five-for-the-future
# Remove build files

Fixes #17 – Removes the `.css` files from the repo. `style-rtl.css` is also removed, but this isn't actually created by the build process, do we need this file? If so, I'll add `grunt-rtlcss` to generate RTL versions of the styles before this gets merged.

**To test**

- Make a change to a sass file, run `npm run build`
- You should only see the sass file in the changed files.
